calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/CalendarHomePage.java

```diff
@@ -163,12 +164,12 @@ public void goToRightMenuTaskEventFromWeekView(String name, selectDayOption opti
     }
   }
 
-  /**
-   * goToRightMenuTaskEventFromMonthView
-   *
-   * @param name name of event or task
-   * @param date date of event: format (MMM dd yyyy)
-   */
+    /**
+     * goToRightMenuTaskEventFromMonthView
+     *
+     * @param name name of event or task
+     * @param date date of event: format (MMM dd yyyy)
+     */
```

Comment:
Useless change. Indentation was good.

Reply:
Identation is good.
![Screenshot from 2019-06-26 17-12-01](https://user-images.githubusercontent.com/50202793/60192034-8dba5b80-9835-11e9-9957-52746e7311e0.png)

Reply:
As your screenshot show, the documentation block is not aligned with the method. The change must be reverted, it was ok before.